python/spacewalk/satellite_tools/spacewalk-repo-sync

```diff
@@ -69,66 +69,67 @@ def main():
 
     global LOCK
     try:
-        LOCK = rhnLockfile.Lockfile('/run/spacewalk-repo-sync.pid')
-    except rhnLockfile.LockfileLockedException:
-        systemExit(1, "ERROR: attempting to run more than one instance of "
-                      "spacewalk-repo-sync Exiting.")
-
-    parser = OptionParser()
-    parser.add_option('-l', '--list', action='store_true', dest='list',
+        parser = OptionParser()
```

Comment:
I wouldn't move the option parsing inside the `try` clause, as the only exception we are really catching in the "except" part is related to the file locking and not to the option parsing.

I would simply move the entire "try/except" block after the option parsing part.